src/le.py

```diff
@@ -2577,6 +2644,9 @@ def process_params(self, params):
                 self.pull_server_side_config = value == "True"
             elif name == "--datahub":
                 self.set_datahub_settings(value)
+            elif name == "--multilog":
+                # multilog is only True if pathname is good
```

Comment:
Well technically it can only EVER be true since the below function never returns any value other then True